spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cassandra/CassandraProperties.java

```diff
@@ -26,12 +26,14 @@
 import com.datastax.driver.core.policies.RetryPolicy;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.data.cassandra.config.SchemaAction;
```

Comment:
That's not storing the `String` value but the enum so the general cassandra auto-config has no a direct dependency to spring data. I thought we discussed that already. Am I missing something?